styleguide/_themes/global/scss/fonts.scss

```diff
@@ -159,3 +172,24 @@ i {
 .copyright {
   font-size: .75em;
 }
+
+blockquote {
+  border-left: 0;
+  font-size: 1.75rem;
```

Comment:
As far as a definitive answer, I think we have to reflect back on the maintainable css model we're trying to follow. Basically it mandates that each chunk of code can stand in isolation. By nature, rem is always relative to the root font size. Em is always relative to itself or its closest ancestor. 

If we ever find ourselves with nested HTML modules for whatever reason, the parent module probably shouldn't have any say in the child module's design, as we should be able to move it anywhere else in the HTML structure (including outside the parent module) and it still will look exactly the same. 

`rem` gives us that behavior, while `em` does not, so that would be my recommendation. 

I did find an interesting post on the subject:  http://zellwk.com/blog/rem-vs-em/

It does contradict my recommendation of picking a single unit, but that said, we're not really talking about padding/margins here so not all of it applies.  It is a very interesting read though.